wordpress-mobile/gutenberg-mobile
# Fix potential crash after "uuid" package upgrade in Gutenberg

Fixes potential crash after `uuid` package upgrade in Gutenberg:
https://github.com/wordpress/gutenberg/pull/21258

The new `uuid` version uses `crypto.getRandomValues` which is not supported in React Native:
https://github.com/uuidjs/uuid/tree/f6ce4bffc2b10d92f3164e76e877e5cdb7942474#getrandomvalues-not-supported

To test:

- Start Android and iOS apps
- Apps should not crash on startup
- Select "Debug" from in-app menu
- Apps should not crash

PR submission checklist:
